### DISCUSSION TITLE
Nestjs, mikro-orm and postgres can't generate table automatically

### DISCUSSION THREAD

**Discussion by kaiwenxiao** [mikro-orm/mikro-orm#5173] [General]
### Describe the bug

Could you inspect the config? Thanks. I'm newer for Postgres and MikroOrm, weird to debugge this. I think table generated automatically will be decent, like TypeOrm does. Thanks. By the way, I'm Use Postgres16 and navicat16(which a visaulization tool of SQL)

### Reproduction

Repository: https://github.com/kaiwenxiao/advance-nest

Some important captures that may help:
![7dc476b9709cbd748e1009678111fb8](https://github.com/mikro-orm/mikro-orm/assets/31737119/db3a9cc0-49f4-4483-bd04-b73d7f1be5e3)

![714a72f06014f4c797905e0511134c0](https://github.com/mikro-orm/mikro-orm/assets/31737119/90a81223-6e25-4eb2-833b-f54bcf6b258f)

![c82ab008b24be1bf3814e880efb898b](https://github.com/mikro-orm/mikro-orm/assets/31737119/92c50b78-77ab-4a1c-b5b1-b65f1b4acfd0)

![f3ad5a293c40f10c93ea0be4f841a0f](https://github.com/mikro-orm/mikro-orm/assets/31737119/dbdbe6a6-1d6a-4fc8-be5b-434fadc0fcbc)

### What driver are you using?

None

### MikroORM version

5

### Node.js version

20

### Operating system

Windows11

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Slack](https://join.slack.com/t/mikroorm/shared_invite/enQtNTM1ODYzMzM4MDk3LWM4ZDExMjU5ZDhmNjA2MmM3MWMwZmExNjhhNDdiYTMwNWM0MGY5ZTE3ZjkyZTMzOWExNDgyYmMzNDE1NDI5NjA).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
https://mikro-orm.io/docs/faq#how-can-i-synchronize-my-database-schema-with-the-entities
